frontend/components/campaigns/campaignCharities/CampaignCharityCard.tsx

```diff
@@ -24,15 +30,28 @@ const CampaignCharityCard = ({ campaignCharity, onClick }: Props) => {
       </Grid>
 
       <Grid item xs={10}>
-        <Stack paddingTop={1} paddingBottom={1} paddingRight={1}>
-          <Typography variant="h4">{campaignCharity.charity.name}</Typography>
+        <Stack direction="row" spacing={-1} sx={charityGraphInfoStackSx}>
```

Comment:
```suggestion
        <Stack direction="row" sx={charityGraphInfoStackSx}>
```
If we adopt the width approach